main.go

```diff
@@ -179,6 +180,30 @@ func main() {
 		os.Exit(1)
 	}
 
+	decoder := admission.NewDecoder(mgr.GetScheme())
+
+	var autoAnnotationConfig auto.AnnotationConfig
+	var autoAnnotationMutator *auto.AnnotationMutators
```

Comment:
nit: These don't need to be declared outside of the block. They are not used anywhere else.

Reply:
updated